<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpAccelerationDialog</class>
 <widget class="QDialog" name="HelpAccelerationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Help for acceleration</string>
  </property>
  <widget class="QLabel" name="labelInfo1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>441</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string>1. First you need to calibrate device 
for reliable results. To calibrate the 
device, please mount the device securely 
e.g. to your cars dashboard. The device 
must be oriented so that the device's 
backside is pointing towards your driving 
direction. Then press calibrate button and 
wait while calibrating is done. After that 
do not change device position.</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCalibratePic">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>130</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="graphics.qrc">:/new/prefix1/Graphics/Speedometer_calibrate.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNext">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>320</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNext">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>290</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLast">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>320</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLast">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>290</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Last</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfo2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>-10</y>
     <width>441</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string>2. Choose category from list 
which you want to complete. 

3. Press start button for starting measuring. 
Measuring view opens and then all you need to do is to accelerate 
your car to the speed you have choosed.</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStartPic">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>130</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="graphics.qrc">:/new/prefix1/Graphics/Speedometer_start.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfo3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>441</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string>4. Measuring view shows your current 
speed and time elapsed. When you 
have reached the target speed view 
automatic switch to the result view.

If you want to stop measuring before the 
target speed has been reached push Abort 
button and you'll get back to the acceleration 
start view.</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMeasurePic">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>50</y>
     <width>241</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="graphics.qrc">:/new/prefix1/Graphics/measuringDialog.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfo4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>441</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string>5. Result view shows information about 
your acceleration.

6. Push Send to server -button for sending 
your time to the server. You have to 
be log in for that.

7. Push New run -button for start 
measuring at beginning.</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="graphics.qrc"/>
 </resources>
 <connections/>
</ui>
